README.md

```diff
@@ -30,14 +30,29 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. Describe the biggest difference between `.forEach` & `.map`.
 
+.map creates a new array with the function affecting the elements, while .forEach mutates the original array.
+
 2. What is the difference between a function and a method?
 
+they are both functions but a method is function that is a property of an object.
+
 3. What is closure?
 
+Closure gives you access to an outer function’s scope from an inner function.
+
 4. Describe the four rules of the 'this' keyword.
 
+```
+    1. Window Binding - When in the global scope, the value of “this” will be the window/console Object
+    2. Implicit Binding - object before the . is 'this'
+    3. New Binding - when a constructor function is used.
+    4. Explicit Binding - when call or apply method is used ```this``` is explicitly defined.
+```
+
 5. Why do we need super() in an extended class?
 
+super() is used to tell a parent’s constructor to be concerned with the child’s attributes
```

Comment:
What specifically happens so that the parent constructor is concerned with the child's attributes?

Reply:
If I am reading your question right, It calls the methods and properties of the parent class for the child to use.

Reply:
precisely, although it would be more rather it calls the properties of the parents and then assigns the prototype methods from the parent to the child